doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md

```diff
@@ -5,4 +5,162 @@ permalink: /:path/
+    * `Devel/make`
+    * `Devel/gcc-core`
+    * `Devel/gcc-g++`
+    * `Devel/mingw-gcc-core`
+    * `Devel/mingw-gcc-g++`
+    * `Devel/mingw-runtime`
+
+
+    Refer to [SO question page](http://superuser.com/questions/304541/how-to-install-new-packages-on-cygwin) for details how to install packages on Cygwin.
+1. Install the GNU ARM toolchain: [gcc-arm-none-eabi](https://launchpad.net/gcc-arm-embedded)
+to the `opt\kaa` directory under the Cygwin root directory (default is `C:\cygwin`)
+1. Install [CC3200SDK](http://www.ti.com/tool/cc3200sdk) to the `opt\kaa`
+(if the directory doesn't exist, create it) directory under the Cygwin root directory.
+1. Install [cmake](http://www.cmake.org/) and add it bin directory to the system environment.
+
+To enable debugging for your CC3200 applications, you will also need to build OpenOCD as
+described in [the official CC3200-Getting_Started_Guide][cc3200-getting-started-guide] (item 3.3.3)
+
+For more information, please refer to the official [CC3200-Getting_Started_Guide][cc3200-getting-started-guide]
+
+[cc3200-getting-started-guide]: http://www.ti.com/lit/ug/swru376d/swru376d.pdf
+
+
+# Creating applications based on C SDK
+
+## Building C SDK for Linux
+
+Before creating applications based on the C SDK, you should obtain the C SDK and build a static
+library from it. To do so, generate the [C SDK in Admin UI]({{root_url}}/Administration-guide/Tenants-and-applications-management/#generating-endpoint-sdk),
+then extract the archive, cd to it and execute the following for Linux:
+
+```
+mkdir -p build
+cd build
+cmake -DKAA_PLATFORM=cc32xx -DCMAKE_TOOLCHAIN_FILE=../toolchains/cc32xx.cmake ..
+make
+```
+
+Refer to [C SDK Linux page]({{root_url}}/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/) for more details.
+
+## Building C SDK for Windows
+
+Open the Cygwin terminal and execute the following:
+
+```
+mkdir -p build
+cd build
+cmake.exe -G "Unix Makefiles" -DKAA_PLATFORM=cc32xx -DKAA_TOOLCHAIN_PATH=c:/cygwin/opt/kaa -DCMAKE_TOOLCHAIN_FILE=../toolchains/cc32xx.cmake ..
+make
+```
+
+# Example
+
+To quickly start with the Kaa IoT platform, you can download one of the Kaa demo applications from the [Kaa Sandbox]({{root_url}}/Getting-started/) and run it on the CC3200 board. We recommend you to start with the ConfigurationDemo.
+
+Connect CC3200 LaunchPad to your PC through a micro-USB connector and execute the following in your terminal:
+
+```
+tar -xzf configuration_demo.tar.gz
+cd CConfigurationDemo
+./build.sh build
+```
+
+To launch the application, execute the following:
+
+```
+/opt/kaa/gcc-arm-none-eabi/bin/arm-none-eabi-gdb -x /opt/kaa/cc3200-sdk/tools/gcc_scripts/gdbinit build/demo_client.afx
+```
+
+NOTE: If you want to see the debug output in the terminal, make sure to get connected to `/dev/ttyUSB{0,1}.`
+
+# Flashing (Windows only)
+
+Jumpers on the CC3200 board should be connected as shown below.
+
+![Jumpers](attach/jumpers_debug_mode_400.png)
```

Comment:
I can't see the `attach/` folder with pin-out diagram. Have you added it to index?